### PR TITLE
test: Ensure agent client starts

### PR DESCRIPTION
**Pull request by jesse-c**
**What this PR does / why we need it**:

I've also added some comments based on my understanding of the code. Along with this, I've added another assertion to the tests, and also made the code more readable.

**Which issue(s) this PR fixes**:

N/A

**Special notes for your reviewer**:

The readability changes are purely _imho_ and I'm not beholden to them.

### CODE REVIEW

File: scheduler/cmd/scheduler/main.go [hunk 1]
```diff
@@ -24,19 +24,19 @@ import (
 	"time"
 
 	"github.com/envoyproxy/go-control-plane/pkg/cache/v3"
-	serverv3 "github.com/envoyproxy/go-control-plane/pkg/server/v3"
+	envoyServerControlPlaneV3 "github.com/envoyproxy/go-control-plane/pkg/server/v3"
```
Comment (jesse-c) [inline scheduler/cmd/scheduler/main.go:27]: **explanation**: The numbering was confusing

File: scheduler/cmd/scheduler/main.go [hunk 2]
```diff
@@ -228,25 +228,28 @@ func main() {
...
 	err = s.StartGrpcServers(allowPlaintxt, schedulerPort, schedulerMtlsPort)
 	if err != nil {
-		log.WithError(err).Fatalf("Scheduler start servers error")
+		log.WithError(err).Fatalf("Failed to start server gRPC servers")
 	}
 
 	err = as.StartGrpcServer(allowPlaintxt, agentPort, agentMtlsPort)
 	if err != nil {
-		log.Fatalf("Failed to start agent grpc server %s", err.Error())
+		log.WithError(err).Fatalf("Failed to start agent gRPC server")
```
Comment (jesse-c) [inline scheduler/cmd/scheduler/main.go:239]: **explanation**: Everywhere else used `WithError`

File: scheduler/docs/design.md
Comment (jesse-c) [inline scheduler/docs/design.md:1]: **explanation**: A mix of spelling and grammatical improvements

File: scheduler/pkg/agent/client.go [hunk 3]
```diff
@@ -677,8 +679,9 @@ func (c *Client) sendModelEventError(
 	event agent.ModelEventMessage_Event,
 	err error,
 ) {
+	c.logger.WithError(err).Errorf("Failed to load model, sending error to scheduler")
```
Comment (jesse-c) [inline scheduler/pkg/agent/client.go:682]: **explanation**: Extra verbose logging that was helpful to me when investigating an issue

File: scheduler/pkg/agent/repository/triton/triton_test.go
Comment (jesse-c) [inline scheduler/pkg/agent/repository/triton/triton_test.go:1]: **explanation**: This change, and the similar ones below, were reported by `structcheck`

**Review by cliveseldon — APPROVED** [review]

**Pull request merged**